lib/common-utils.ts

```diff
@@ -72,21 +72,23 @@ export const determineFinalStatus = (statuses: TestStatus[]): TestStatus | null
 
 export const getUrlWithBase = (url: string | undefined, base: string | undefined): string => {
     try {
+        if (isEmpty(base)) {
```

Comment:
It looks like that we can write this code outside `try` statement, because i don't see what can trigger an error in this code.